### ISSUE TITLE
Assess top tasks for self-service optimization - 21st Century IDEA Act

### ISSUE THREAD

**Issue by Maroyafaied** [DOI-ONRR/onrr.gov-site#2907]
Within 180 days, agencies that provide services to the public shall review the highest volume public-facing services and identify the tasks required to access each service as well as the channels available to complete each task (e.g., online, phone, mail, in person). Agencies shall provide to OMB a list of opportunities for no less than 5 top tasks that can be newly designed or further optimized as self-service digital options.

**Comment by AlexandracmONRR** [comment]
From previous issue:

- [ ] review available web metrics to identify top pages for common searches; and
- [ ] conduct a high level review of potentially duplicative or substantively similar content across websites within the agency or across agencies.
- [ ] Agencies shall provide to OMB a list, in aggregate across channels, of the 25 to 50 most commonly asked questions and a list of no less than 10 opportunities to retire, consolidate, or explain duplicative web content; redesign or rewrite underperforming web content; or further optimize web content to improve search

Additional notes:

- [ ] align with user types
- [ ] review previous work to see if any similar work was done last year


Other notes: 
- should be at least 25 tasks

**Comment by AlexandracmONRR** [comment]
Assess top tasks for self-service optimization: Within 180 days, agencies that provide services to the public shall review the highest volume public-facing services and identify the tasks required to access each service as well as the channels available to complete each task (e.g., online, phone, mail, in person). Agencies shall provide to OMB a list of opportunities for no less than 5 top tasks that can be newly designed or further optimized as self-service digital options.

**Comment by AlexandracmONRR** [comment]
**NRRD:**
https://doimspp.sharepoint.com/:x:/r/sites/NRRD/_layouts/15/Doc.aspx?sourcedoc=%7BC7B1AB86-5B3E-43A9-B003-3A6B8975EE53%7D&file=Copy%20of%20onnr%20spreadsheet%20edit%20.xlsx&action=default&mobileredirect=true
**ONRR:**
https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/ONRR.gov/User[…]?d=waa698db3e8f9428ca8684e7018b320a5&csf=1&web=1&e=7F1aHh

(There may be an additional spreadsheet for ONRR that discusses user types and their tasks that we can use.)

**Comment by eelzi-ONRR** [comment]
@AlexandracmONRR I put the analytics in the Digital Experiences folder: https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/Admin/Digital%20Experience/Analytics%20for%20DIgital%20Services%20Top%20Tasks%20-%20FY23.xlsx?d=w16e2207e45444341a3404afcc61bde50&csf=1&web=1&e=djdg4D

It includes top pages views for each site, top downloads and what was the landing page that got the user to the download, and top outbound links to ONRR systems and the referring page (for onrr.gov) and the top events for the top pages (NRRD).  These made the most sense to me to help determine tasks and the paths to do them. All are for FY23

**Comment by eelzi-ONRR** [comment]
@AlexandracmONRR @Maroyafaied I've identified the top 10 tasks for onrr.gov using user research and analytics, and some initial thoughts on what needs to be done to increase self-service optimization. Let me know if you have questions or were expecting this in a different format. One of the tabs is the top tasks for this assignment, and I kept the other tab of all the tasks I pulled form user research in there so we have it for reference. I also put this sheet in the user research folder.

https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/Admin/Digital%20Experience/ONRR%20User%20Tasks.xlsx?d=w99126021e41943f8a9e82e731c5cbc92&csf=1&web=1&e=yk8Gkw

**Comment by Maroyafaied** [comment]
@eelzi-ONRR this is great, I added a column for "user types involved" and realized we didn't have any tasks for the user type "ONRR's Partners" so I added a row (missing info for columns D & E). I added a couple of comments suggesting combining some tasks, let me know your thoughts or if any comments don't make sense. Great job.

**Comment by AlexandracmONRR** [comment]
@Maroyafaied and @eelzi-ONRR - I've added a tab within the excel Erin created for NRRD specific optimization tasks. These are based on the user tasks spreadsheet that was previous made and the analytics that Erin pulled last week. I still have some work to do on these, but if you can take a quick look, you can let me know if I'm going the right direction or off base. Thank you!

**Comment by Maroyafaied** [comment]
@eelzi-ONRR I looked at the tab "ONRR-overall tasks" here's a summary of my recommendations:
1. Combine "File royalty report(2014) and "Access Reporting tools" into one task "Access reporting systems to file reports and make payments". 
2. Combine "Cross-reference lease & agreement number" & "look codes" into one task " Access reference lists & codes". For the column what it entails would be 
3. Combine "onboard new employees" and "train new reporters" into one task "Train new employees or reporters"
4. Take royalty reporting out from  "Help industry troubleshoot royalty reporting" 
5. Change "Submit or change official contact (4444)" to "find or update contact information"
6. Add the following 3 tasks: "Find events and information to share", "Look up unbundling cost allocations", "understand lease valuation".

I added a tab in the excel sheet "MF onrr.gov tasks"  that summarizes my recommendations.  Under each task, I put items from the overall tasks and the 2nd column has what the task entails.   Let me know if a call would be easier. Thanks
